packages/material-ui-lab/src/TreeItem/TreeItem.d.ts

```diff
@@ -20,10 +20,18 @@ export interface TreeItemProps
    * The icon to display next to the tree node's label.
    */
   icon?: React.ReactNode;
+  /**
+   * `onClick` handler for the icon container. Call `event.preventDefault()` to prevent firing on `onNodeToggle`.
```

Comment:
```suggestion
   * `onClick` handler for the icon container. Call `event.preventDefault()` to prevent `onNodeToggle` from being called.
```
? Same for onLabelClick

Reply:
That's better!